schemas/json/text-resources/text-resources.schema.v1.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$id": "https://altinncdn.no/schemas/json/text-resources/text-resources.schema.v1.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Altinn text resource file",
+  "description": "Schema that describes a set of text resources for Altinn applications.",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "title": "Schema",
+      "description": "The schema that this file adheres to"
+    },
+    "language": {
+      "type": "string",
+      "title": "Language",
+      "description": "The language of these text resources. Must match the file name, using ISO 639-1 two letter language codes.",
+      "pattern": "^[a-z]{2}$"
+    },
+    "resources": {
+      "type": "array",
+      "title": "Text resources",
+      "description": "A list of text resources",
+      "items": { "$ref": "#/definitions/resource" },
+      "additionalItems": { "$ref": "#/definitions/resource" }
```

Comment:
Just curious, why is `additionalItems` necessary?

Reply:
Uhh, not sure. Maybe it's not! I'll test and remove it. It was just a case of throwing definitions at it until something works at one point.